app/_locales/en/messages.json

```
@@ -933,6 +933,22 @@
   "enableAutoDetect": {
     "message": " Enable Autodetect"
   },
+  "enableEIP1559V2": {
+    "message": "Enable Enhanced Gas Fee UI"
+  },
+  "enableEIP1559V2AlertMessage": {
+    "message": "We've updated how gas fee estimation and customization works."
+  },
+  "enableEIP1559V2ButtonText": {
+    "message": "Turn on EIP-1559 V2 in Settings"
```

We should probably use the same term in this button text as well:
```suggestion
    "message": "Turn on Enhanced Gas Fee UI in Settings"
```
This is the last place "EIP-1559 V2" is used in text, so I would not expect anyone to know what it means